### DISCUSSION TITLE
Airdump showing many BSSID for one router

### DISCUSSION THREAD

**Discussion by chauquang9** [aircrack-ng/aircrack-ng#2607] [Q&A]
Hi

![Screenshot from 2024-04-21 20-27-23](https://github.com/aircrack-ng/aircrack-ng/assets/24238273/e6eee450-6c2e-4fcc-a9a3-7eb6c9e7981d)

as my title, I have problem about airdump because I saw neighbor wifi just showing 2.4ghz and 5ghz with one BSSID and ESSI (Thuan Ky), so for example, I sent image above, "Deco E4" and "Deco E4 5Ghz" are showing many different bssid, so I would try to run "sudo aireplay-ng" to try ddos but it's not working with many bssid, so do you guys know how to detect which is primary to try?

Thanks

**Comment by Mister-X-** [comment]
I'm gonna close this because you're doing DoS.

However, for anybody else wondering, you can have the same ESSID with multiple BSSID. That is totally normal.
